P-AXIS_RCS_AUTOPILOT.s

```diff
@@ -347,7 +347,7 @@ R-RATE		INCR	NRTRAPS
 		TS	OMEGAR
 
 # END OF RATE DERIVATION
-#	BEGIN OFFSET ESTIMATER
+#	BEGIN OFFSET ESTIMATOR
```

Comment:
`ESTIMATER` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1427.jpg)